src/Command/AbstractDependencyGraphCommand.php

```diff
@@ -29,6 +31,16 @@ abstract class AbstractDependencyGraphCommand extends Command
      */
     protected $io;
 
+    /**
+     * @var LoggerInterface
+     */
+    protected $logger;
+
+    public function __construct()
+    {
+        parent::__construct();
```

Comment:
Constructor only calling parent. Not very useful :)

Reply:
Ahh, I added that to add an instance of the logger but found out that it needed the input. Forgot to remove it.